scripts/ec2/networking.sh

```diff
@@ -29,13 +29,13 @@ readonly EC2_FILES='/var/tmp/ec2'
 
 [[ -d $EC2_FILES ]] || mkdir -p $EC2_FILES
 
-# The version 3.1.2 is currently the recommended version.
-SRIOV_DRIVER='ixgbevf-3.1.2.tar.gz'
+# The version 4.2.1 is currently the recommended version.
```

Comment:
Okay, just grabbed the source for 4.2.1 and confirmed that `InterruptThrottleRate` is still a valid parameter via the README so we should be good to go here.   Let me know if you'd like me to do further testing, otherwise I'll merge.   Thanks!